### DISCUSSION TITLE
Make history search update automatically

### DISCUSSION THREAD

**Discussion by nonua** [Chatterino/chatterino2#3850] [Ideas]
Hi!
Search history is great, but it lacks automatic updates when new matching messages come in.
It would be nice if, while the window is opened the messages keep pilling up, much like the usercard does.
Is the current behavior actually the feature? What do you think?

**Comment by Mm2PL** [comment]
IMO this is unwanted behaviour. Either we:
  1. make the search popup both add new messages and remove old ones, having it only add new messages introduces issues. It will keep all the messages it was ever shown in memory therefore it's not practical. Let's hope that one user won't be able to spam so much that this becomes an issue with usercards.
  2. make it not update at all

I prefer 2 because I dislike searching for something in a fast-scrolling chat and then having it  disappear.

If you want an updating split with messages which match certain criteria, please use filters[https://wiki.chatterino.com/Filters/].

3 lip 2022 15:40:18 Nonua ***@***.***>:

> Hi!
> Search history is great, but it lacks automatic updates when new matching messages come in.
> It would be nice if, while the window is opened the messages keep pilling up, much like the usercard does.
> Is the current behavior actually the feature? What do you think?

**Reply by nonua** [reply]
I understand the issue you are addressing, fast chat can be a real pain in the butt and I use search mostly for them.
Filters are great but unfortunetly not practical enough to just glance at something...

What about letting the choice to the user to pause the incoming message that meet the search, with a button or shortcut like chat pause ? The default behavior could be added as a setting also
